analytics_data_api/v0/views/learners.py

```diff
@@ -186,11 +215,20 @@ def _validate_query_params(self):
                 if page_size > self.max_paginate_by or page_size < 1:
                     raise ParameterValueError('Page size must be in the range [1, {}]'.format(self.max_paginate_by))
 
+    def list(self, request, *args, **kwargs):
+        """
+        Adds the last_updated field to the results.
+        """
+        response = super(LearnerListView, self).list(request, args, kwargs)
+        last_updated = self.get_last_updated()
+        for result in response.data['results']:
+            result.update(last_updated)
```

Comment:
This differs from the current API spec, where `last_updated` is a key in the top-level JSON object. However, I don't really mind this implementation, and it makes things more consistent. Would you mind updating the API spec?